scripts/firebase-config.ts

```diff
@@ -0,0 +1,58 @@
+import * as fs from 'fs';
+import * as path from 'path';
+import { cert, initializeApp } from 'firebase-admin/app';
+import { getFirestore } from 'firebase-admin/firestore';
+
+const SERVICE_ACCOUNT_PROD = 'serviceAccountProd.json';
+const SERVICE_ACCOUNT_DEV = 'serviceAccount.json';
+const DATABASE_URL_PROD = 'https://cornell-courseplan.firebaseio.com';
+const DATABASE_URL_DEV = 'https://cornelldti-courseplan-dev.firebaseio.com';
+
+function getDatabase(serviceAccount: string, databaseURL: string, appName?: string) {
+  return () => {
+    const credential = cert(serviceAccount);
+    const app = initializeApp(
+      {
+        credential,
+        databaseURL,
+      },
+      appName
+    );
+    return getFirestore(app);
+  };
+}
+
+export const getProdDb = getDatabase(SERVICE_ACCOUNT_PROD, DATABASE_URL_PROD, 'prod');
+export const getDevDb = getDatabase(SERVICE_ACCOUNT_DEV, DATABASE_URL_DEV, 'dev');
+
+const isProd = process.env.PROD === 'true';
```

Comment:
it seems like we're sorta mixing firebase-admin stuff for running locally with running this automatically and checking process.env. It might be helpful to document when some of this can be used, because something like my copy user data script (and I assume a migration script) will probably be run locally and most likely own't have anything in process. it looks like the top part of the file with getDatabase deals with the local stuff and the rest is for a process, but I'm worried it could be confusing having this all in one file.